thoth-export-server/src/xml/onix3_thoth.rs

```diff
@@ -0,0 +1,3368 @@
+                        write_element_block("Subject", w, |w| {
+                            if is_main_subject {
+                                // MainSubject is an empty element
+                                write_element_block("MainSubject", w, |_w| Ok(()))?;
+                            }
+                            XmlElement::<Onix3Thoth>::xml_element(&subject.subject_type, w)?;
+                            match subject.subject_type {
+                                SubjectType::KEYWORD | SubjectType::CUSTOM => {
+                                    write_element_block("SubjectHeadingText", w, |w| {
+                                        w.write(XmlEvent::Characters(&subject.subject_code))
+                                            .map_err(|e| e.into())
+                                    })
+                                }
+                                _ => write_element_block("SubjectCode", w, |w| {
+                                    w.write(XmlEvent::Characters(&subject.subject_code))
+                                        .map_err(|e| e.into())
+                                }),
+                            }
+                        })?;
+                        if is_main_subject {
+                            main_subject_found.push(subject.subject_type.clone());
+                        }
+                    }
+                    write_element_block("Audience", w, |w| {
+                        // 01 ONIX audience codes
+                        write_element_block("AudienceCodeType", w, |w| {
+                            w.write(XmlEvent::Characters("01")).map_err(|e| e.into())
+                        })?;
+                        // 06 Professional and scholarly
+                        write_element_block("AudienceCodeValue", w, |w| {
+                            w.write(XmlEvent::Characters("06")).map_err(|e| e.into())
+                        })
+                    })
+                })?;
+                write_element_block("CollateralDetail", w, |w| {
+                    if let Some(mut short_abstract) = self.short_abstract.clone() {
+                        // Short description field may not exceed 350 characters
+                        short_abstract.truncate(350);
+                        write_element_block("TextContent", w, |w| {
+                            // 02 Short description
+                            write_element_block("TextType", w, |w| {
+                                w.write(XmlEvent::Characters("02")).map_err(|e| e.into())
+                            })?;
+                            // 00 Unrestricted
+                            write_element_block("ContentAudience", w, |w| {
+                                w.write(XmlEvent::Characters("00")).map_err(|e| e.into())
+                            })?;
+                            write_full_element_block(
+                                "Text",
+                                Some(vec![("language", "eng")]),
```

Comment:
The language attribute seems to be optional (unless you include multiple <Text> elements within a <TextContent>), so let's not hardcode it